infra/core/storage/storage-account.bicep

```diff
@@ -73,9 +73,6 @@ param sku object = { name: 'Standard_LRS' }
 @description('Determines whether or not trusted azure services are allowed to connect to this account')
 param bypass string = 'AzureServices'
 
-@description('The firewall rules to install on the sql-server.')
-param firewallRules FirewallRules?
```

Comment:
It looks like the FirewallRules type is unused now. If so, we should remove it from this file.